src/ClpCInterface.jl

```diff
@@ -418,12 +420,36 @@ function add_rows(model::ClpModel, number::Integer, row_lower::Vector{Float64},
     @clp_ccall addRows Void (Ptr{Void}, Int32, Ptr{Float64}, Ptr{Float64}, Ptr{Int32}, Ptr{Int32}, Ptr{Float64}) model.p number row_lower row_upper row_starts columns elements
 end
 
+#This function exists in cpp but not c interface
+function add_row(model::ClpModel, number_in_row::Integer, columns::Vector{Int32}, elements::Vector{Float64},
+        row_lower::Float64, row_upper::Float64)
+        row_starts_vector = Vector{Int32}(2)
+        row_starts_vector[1] = 0
+        row_starts_vector[2] = number_in_row
+        row_upper_vector = [row_upper]
+        row_lower_vector = [row_lower]
+        add_rows(model, 1, row_lower_vector, row_upper_vector, row_starts_vector, columns, elements)
```

Comment:
Isn't it just as simple to go
`add_rows(model, 1, [row_lower], [row_upper], [0, number_in_row], columns, elements)`